<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="lblPortName">
              <property name="text">
               <string>串口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbPortName">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lblBaudRte">
              <property name="text">
               <string>波特率</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbBaudRate">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lblDataBits">
              <property name="text">
               <string>数据位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbDataBits">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="lblParity">
              <property name="text">
               <string>奇偶校验位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbParity">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="lblStopBits">
              <property name="text">
               <string>停止位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbStopBits">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="lblFlowControl">
              <property name="text">
               <string>流控制</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbFlowControl">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>无</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>数据接收区</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QTextEdit" name="teReceiveData"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>数据发送区</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QTextEdit" name="teSendData"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnOpen">
          <property name="text">
           <string>打开串口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSend">
          <property name="text">
           <string>发送数据</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
